Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -322,6 +322,14 @@ final class UIViewExtensionsTests: XCTestCase {
         XCTAssertEqual(view1.transform, view3.transform)
     }
 
+    #if os(iOS) // UITableViewCell nib only works on iOS
```

Comment:
You can create a separate XIB file for tvOS. You may need to create a separate folder for tvOS-specific resources so that you can use the same file name.

Reply:
This is a good idea. I'm slightly inclined to save it for the next PR since it's a minor testing improvement, though. What do you think?

Reply:
That sounds like one of those `TODO`s that will exist forevermore in your code 😅

Reply:
You can read me like an open book :)

Happy to do it if you want, though. Give me a nudge either way!

Reply:
Yes, please do it. It should take you 2 minutes

Reply:
I will give it a whirl. I'm not familiar with the project and I've never used tvOS, so I doubt this will take me two minutes.

I think I can create a `UIImageView.xib` for each test target since that view is supported on both platforms. Are you envisioning a `tvOsResourcesTests` folder inside `Tests`?

Reply:
Indeed. In `Test/ResourcesTests` create a `tvOS` folder and create the XIB in there. Remember to make sure that you select the `tvOS` section when creating the XIB file so that it will load correctly.

![Screen Shot 2020-07-28 at 17 59 07](https://user-images.githubusercontent.com/924764/88690519-0ad09800-d0fc-11ea-96d2-be633c8de749.png)

Reply:
That was helpful, thank you. How about this?